src/pages/proposal.css

```diff
@@ -22,10 +22,6 @@
   min-height: 24px;
 }
 
-.ProposalDetailPage__Description img {
```

Comment:
Are you sure this is not being used? (like, an img inside a proposal markdown)

Reply:
Yeap, my bad. Idk why I deleted this. Thanks for the heads up!